tests/pipeline/ai/test_fall_detect.py

```diff
@@ -161,8 +161,7 @@ def sample_callback(image=None, inference_result=None, **kwargs):
 
     assert result
     assert len(result) == 1
-    category, confidence, box, angle = result[0]
-    assert box   # Add this line to avoid 'Unused local variable' 
+    category, confidence, angle = result[0]
```

Comment:
@bhavikapanara Simply removing the box detection makes this test pass but breaks the UI. We need to discuss an event data format that the edge and UI can agree on. Until fall detection, all inferences had the same output format. Now that we are introducing keypoints and angles as opposed to boxes, we need to rethink how the edge and UI agree on detection format.

Reply:
@ivelin ok...then let's first finalize output format. Next, will modify the code accordingly.

Reply:
@bhavikapanara Here is how we will proceed here. Instead of redefining the event format, we will populate a bounding box value by calculating the box surrounding all detected keypoins. See [this code](https://github.com/tensorflow/tfjs-models/blob/08dd2ed9ffcd3f3604026c2374f99a3498184d33/posenet/src/util.ts#L45) from the posenet js demo.

```
const {NEGATIVE_INFINITY, POSITIVE_INFINITY} = Number;
export function getBoundingBox(keypoints: Keypoint[]):
    {maxX: number, maxY: number, minX: number, minY: number} {
  return keypoints.reduce(({maxX, maxY, minX, minY}, {position: {x, y}}) => {
    return {
      maxX: Math.max(maxX, x),
      maxY: Math.max(maxY, y),
      minX: Math.min(minX, x),
      minY: Math.min(minY, y)
    };
  }, {
    maxX: NEGATIVE_INFINITY,
    maxY: NEGATIVE_INFINITY,
    minX: POSITIVE_INFINITY,
    minY: POSITIVE_INFINITY
  });
}
```

This will resolve the immediate issue with format incompatibility in a meaningful way.

We should still add keypoints as an additional event attribute, which will be logged to disk and used by the UI to render overlay of a pose skeleton.

Reply:
@ivelin `Instead of redefining the event format, we will populate a bounding box value by calculating the box surrounding all detected keypoins.`

Didn't get it exactly what I need to do here by calculating the box surrounding all detected keypoitns.?

Reply:
Proceed with a reasonable implementation of a bounding box. That will show a meaningful output in the current UI. A later version of the UI will also use the inference keypoints to draw a stick figure.